Kickstarter-iOS/DataSources/ProfileDataSource.swift

```diff
@@ -7,13 +7,6 @@ internal final class ProfileDataSource: ValueCellDataSource {
 
   internal enum Section: Int {
     case projects
-    case emptyState
-  }
-
-  internal func emptyState(visible visible: Bool) {
-//    self.set(values: visible ? [()] : [],
-//             cellClass: ProfileEmptyStateCell.self,
-//             inSection: Section.emptyState.rawValue)
```

Comment:
Note: I just removed all references to the Profile empty state since we weren't using it anyway. Will await the redesign.